mlflow/server/js/src/model-registry/components/ModelVersionPage.js

```diff
@@ -142,8 +142,9 @@ export class ModelVersionPageImpl extends React.Component {
               clearInterval(this.pollIntervalId);
               if (Utils.shouldRender404(requests, this.state.criticalInitialRequestIds)) {
                 return (
-                  <Error404View
-                    resourceName={`Model ${modelName} v${version}`}
+                  <ErrorView
+                    statusCode={404}
+                    resourceName={`Model ${modelName} v${version} does not exist`}
```

Comment:
I don't see resourceName prop in ErrorView?

Reply:
Resolved